ci/taos/plugins-staging/pr-nnstreamer-apptest.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+##
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+##
+# @file pr-nnstreamer-apptest.sh
+# @brief Check nnstreamer sample app working well.
+# @see      https://github.com/nnsuite/TAOS-CI
```

Comment:
Please append the below statement
```bash
@see     https://github.com/nnsuite/nnstreamer/wiki/usage-examples-screenshots
```

Reply:
@jaeyun-jung  PTAL, This commit is to automate the example that you worte at the wiki page.